pages/index.js

```diff
@@ -1,30 +1,86 @@
-import React from "react";
+import React, { useState } from "react";
 
-import TodoInput from "../components/todo-input";
+import MemoizedTodoInput from "../components/todo-input";
```

Comment:
```suggestion
import {MemoizedTodoInput} from "../components/todo-input";
```
이렇게 import하셔야 MemoizedTodoInput을 제대로 가져옵니다!
괄호를 쓰지 않으면 이름과 상관 없이 default export된 모듈을 import합니다 ㅎ ㅠ